catalog-rest-service/src/main/resources/ui/src/pages/DatasetDetailsPage/DatasetDetailsPage.component.tsx

```diff
@@ -57,9 +58,55 @@ import { serviceTypeLogo } from '../../utils/ServiceUtils';
 import { getOwnerFromId, getTierFromTableTags } from '../../utils/TableUtils';
 import { getTableTags } from '../../utils/TagsUtils';
 
+const datasetTableTabs = [
+  {
+    name: 'Schema',
+    path: 'schema',
+  },
+  {
+    name: 'Profiler',
+    path: 'profiler',
+  },
+  {
+    name: 'Lineage',
+    path: 'lineage',
+  },
+  {
+    name: 'Manage',
+    path: 'manage',
+  },
+];
+
+export const getCurrentTab = (tab: string) => {
```

Comment:
This should be inside component utils file

Reply:
move to utils file